charts/consul/templates/crd-jwtproviders.yaml

```diff
@@ -111,8 +109,7 @@ spec:
                       cacheDuration:
                         description: "CacheDuration is the duration after which cached
                           keys should be expired. \n Default value is 5 minutes."
-                        format: int64
-                        type: integer
+                        type: string
```

Comment:
yup! these are fields that are Duration fields. The correct type to use for CRDs for durations is metav1.Duration which serializes to string vs time.Duration that serializes to int. this will allow someone to specify `10s` and that get parsed correctly instead of blowing up because `cant use type string for int64`

Reply:
Cool! Main reason I ask is, I was just noticing that some of these seemed to be CRDs currently in use - if that's the case, I was wondering whether we'd need to worry about backwards compatibility for field type changes.

Reply:
From sync conversation: the new duration type in Go defaults to millis as the unit, so this is backwards-compatible.